src/components/common/Loading.tsx

```diff
@@ -0,0 +1,12 @@
+import { Container, Skeleton } from '@mui/material'
+
+const Loading = () => {
+  return (
+    <Container>
+      <Skeleton animation="wave" variant="text" height={100}></Skeleton>
```

Comment:
```suggestion
      <Skeleton variant="text" height={100}></Skeleton>
```
That was actually something I didn't mean to commit